tree/treeplayer/src/TTreeProcessorMT.cxx

```diff
@@ -167,13 +168,30 @@ static ClustersAndEntries MakeClusters(const std::vector<std::string> &treeNames
       std::vector<EntryCluster> clusters;
       while ((start = clusterIter()) < entries) {
          end = clusterIter.GetNextEntry();
-         // Add the current file's offset to start and end to make them (chain) global
-         clusters.emplace_back(EntryCluster{start + offset, end + offset});
+         // Currently, if a user specified a range, the clusters will be only globally obtained
```

Comment:
Thanks for adding these comments, I would still like to see a concrete example with at least 3 files and 3 tasks